basic_system/Cargo.toml

```diff
@@ -30,6 +30,7 @@ rand = {version = "0.9.0", optional = true}
 serde = { version = "1.0", default-features = false, features = ["derive", "alloc"], optional = true }
 strum_macros = "0.27.1"
 paste = "1.0.15"
+zerocopy = { workspace = true }
```

Comment:
But should we actually have it here, and in the workspace generally? Seems like it's not used